src/Storage/Migration/v0.00/03-setup-functions.sql

```diff
@@ -1,4 +1,151 @@
 -- instances ---------------------------------------------------
+CREATE OR REPLACE FUNCTION storage.readinstancefromquery(
+    _appId TEXT DEFAULT NULL,
+    _appIds TEXT[] DEFAULT NULL,
+    _archiveReference TEXT DEFAULT NULL,
+    _continue_idx BIGINT DEFAULT 0,
+    _created_eq TIMESTAMPTZ DEFAULT NULL,
+    _created_gt TIMESTAMPTZ DEFAULT NULL,
+    _created_gte TIMESTAMPTZ DEFAULT NULL,
+    _created_lt TIMESTAMPTZ DEFAULT NULL,
+    _created_lte TIMESTAMPTZ DEFAULT NULL,
+    _dueBefore_eq TEXT DEFAULT NULL,
+    _dueBefore_gt TEXT DEFAULT NULL,
+    _dueBefore_gte TEXT DEFAULT NULL,
+    _dueBefore_lt TEXT DEFAULT NULL,
+    _dueBefore_lte TEXT DEFAULT NULL,
+    _excludeConfirmedBy JSONB[] DEFAULT NULL,
+    _instanceOwner_partyId INTEGER DEFAULT NULL,
+    _instanceOwner_partyIds INTEGER[] DEFAULT NULL,
+    _lastChanged_eq TIMESTAMPTZ DEFAULT NULL,
+    _lastChanged_gt TIMESTAMPTZ DEFAULT NULL,
+    _lastChanged_gte TIMESTAMPTZ DEFAULT NULL,
+    _lastChanged_idx TIMESTAMPTZ DEFAULT NULL,
+    _lastChanged_lt TIMESTAMPTZ DEFAULT NULL,
+    _lastChanged_lte TIMESTAMPTZ DEFAULT NULL,
+    _org TEXT DEFAULT NULL,
+    _process_currentTask TEXT DEFAULT NULL,
+    _process_ended_eq TEXT DEFAULT NULL,
+    _process_ended_gt TEXT DEFAULT NULL,
+    _process_ended_gte TEXT DEFAULT NULL,
+    _process_ended_lt TEXT DEFAULT NULL,
+    _process_ended_lte TEXT DEFAULT NULL,
+    _process_isComplete BOOLEAN DEFAULT NULL,
+    _size INTEGER DEFAULT 100,
+    _sort_ascending BOOLEAN DEFAULT FALSE,
+    _status_isActiveOrSoftDeleted BOOLEAN  DEFAULT NULL,
+    _status_isArchived BOOLEAN DEFAULT NULL,
+    _status_isArchivedOrSoftDeleted BOOLEAN DEFAULT NULL,
+    _status_isHardDeleted BOOLEAN DEFAULT NULL,
+    _status_isSoftDeleted BOOLEAN DEFAULT NULL,
+    _visibleAfter_eq TEXT DEFAULT NULL,
+    _visibleAfter_gt TEXT DEFAULT NULL,
+    _visibleAfter_gte TEXT DEFAULT NULL,
+    _visibleAfter_lt TEXT DEFAULT NULL,
+    _visibleAfter_lte TEXT DEFAULT NULL
+    )
+    RETURNS TABLE (id BIGINT, instance JSONB, element JSONB)
+    LANGUAGE 'plpgsql'
+    
+AS $BODY$
+BEGIN
+    IF _sort_ascending IS NULL THEN
+        _sort_ascending := false;
+    END IF;
+
+    RETURN QUERY
+    WITH filteredInstances AS
```

Comment:
Clean up differences in indentation. Some lines have spaces while other have tabs.

Reply:
Done in PR286